### PR TITLE
[AKS] Remove support for az aks get-os-options command

### PR DESCRIPTION
**Pull request by chengliangli0918**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
Remove support for 'az aks get-os-options' command. 
The API required by this command was removed from AKS API 2024-05-01, 2024-05-02-preview or later. So remove this command before aks-preview extension starts to use 2024-05-02-preview API.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by haitch — APPROVED** [review]

**Comment by yonzhan** [comment]
Please fix CI issues

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+6.0.0b2
+++++++++
+* Remove support for `az aks get-os-options` command.
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:17]:
```suggestion
7.0.0b1
++++++++
* [Breaking Change] Remove support for `az aks get-os-options` command.
```

File: src/aks-preview/HISTORY.rst [hunk 2]
```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
...
+
 6.0.0b1
 ++++++++
 * Remove support for `az aks mesh` egress gateway commands.
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:21]:
```suggestion
* [BREAKING CHANGE]: Remove support for `az aks mesh` egress gateway commands.
```

File: src/aks-preview/setup.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "6.0.0b1"
+VERSION = "6.0.0b2"
```
Comment (FumingZhang) [inline src/aks-preview/setup.py:12]:
```suggestion
VERSION = "7.0.0b1"
```

**Review by FumingZhang — APPROVED** [review]
LGTM

File: src/aks-preview/HISTORY.rst [hunk 4]
```diff
@@ -12,9 +12,13 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+7.0.0b1
+++++++++
+* [Breaking Change]: Remove support for `az aks get-os-options` command.
```
Comment (yanzhudd) [inline src/aks-preview/HISTORY.rst:17]: It is better to keep all history logs consistent
```suggestion
* [BREAKING CHANGE]: Remove support for `az aks get-os-options` command.
```

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**